scripts/tests/calibnet_offline_rpc_check.sh

```diff
@@ -12,6 +12,8 @@ stop_services() {
     for port in "${PORTS[@]}"; do
         fuser -k "$port/tcp" || true
     done
+    # Remove downloaded snapshot file
+    rm -rf "$snapshot"
```

Comment:
It'd be great to make this function fire whenever the script is cancelled, e.g., via SIGINT. As of now, when I ran it a few times, I had lingering processes and snapshot artefacts in this directory.